### ISSUE TITLE
No prompt to choose folder on batch download

### ISSUE THREAD

**Issue by DrSNAFU** [eight04/image-picka#14]
Pressing save just dumps all images into the Downloads folder, or the specified subfolder.  No window opens asking me where to download the batch to.

**Comment by eight04** [comment]
I think there is no way to "choose a folder" for batch download in webextension. Webextension does provide a "save as..." option, but it creates save as dialog for each image.

Currently you can only edit the filename pattern manually in batch download UI, and place them in a subfolder.

**Comment by DrSNAFU** [comment]
Ah, unfortunate.

**Comment by DelKatey** [comment]
Maybe an option to zip for batch downloads?

**Comment by mrandreastoth** [comment]
The problem is that Image Picka assumes it should download to Downloads\Image Picka\{Page Title} when the Downloads\Image Picka portion should be user-definable. This applies not only to batch downloaded images but also double-clicked images. For this Image Picka needs a settings page.

**Comment by alexolog** [comment]
> Maybe an option to zip for batch downloads?

Yes please!

**Comment by eight04** [comment]
> the Downloads\Image Picka portion should be user-definable.

You can change them in the settings. The default download folder should be able to be changed in browser settings.

**Comment by alexolog** [comment]
Going through changing settings back and forth just to save an image set in a non-default location is inconvenient.  The "save as zip" suggestion above coupled with a "save as" sounds like a good alternative.

**Comment by necropower** [comment]
> I think there is no way to "choose a folder" for batch download in webextension. Webextension does provide a "save as..." option, but it creates save as dialog for each image.
> 
> Currently you can only edit the filename pattern manually in batch download UI, and place them in a subfolder.

cant we put a absolute path directory in the future releases of it?

**Comment by eight04** [comment]
No. For security, both Chrome and Firefox had decided that extensions can only download files into the download folder.

The only way to bypass this restriction is to install other native software (e.g. a renaming software which can move files), and let Image Picka call it. However, I think this solution is too complex.

**Comment by alexolog** [comment]
Many extensions allow the user to specify a subfolder under downloads.
